### PR TITLE
Update versions for Compose 1.6.0

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: topics/compose/compose-compatibility-and-versioning.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ which version of Jetpack Compose artifact is used:
 
 | Compose Multiplatform version                                                     | Jetpack Compose version | Jetpack Compose Material3 version |
 |-----------------------------------------------------------------------------------|-------------------------|-----------------------------------|
+| [1.6.0](https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.6.0)   | 1.6.1                   | 1.2.0                             |
```
Comment (igordmn) [inline topics/compose/compose-compatibility-and-versioning.md:193]: The version isn't released yet and  the page doesn't exist. The PR should be merged only after the official release

File: topics/compose/compose-compatibility-and-versioning.md [hunk 2]
```diff
@@ -91,17 +91,12 @@ Multiplatform, there are two options:
 
 | Kotlin version | Compose Multiplatform compiler version |
 |----------------|----------------------------------------|
-| 2.0.0-Beta1    | `1.5.4-dev1-kt2.0.0-Beta1`             |
-| 1.9.22         | `1.5.8`                                |
+| 2.0.0-Beta4    | `1.5.9-kt-2.0.0-Beta4`                 |
+| 1.9.22         | `1.5.8.1`                              |
 | 1.9.21         | `1.5.4`                                |
 | 1.9.20         | `1.5.3`                                |
-| 1.9.20-RC      | `1.5.3-rc01`                           |
```
Comment (igordmn) [inline topics/compose/compose-compatibility-and-versioning.md:98]: I removed old prerelease versions of Kotlin. They not needed for users almost in all cases.

**Comment by igordmn** [comment]
The PR is ready for review

**Review by zamulla — APPROVED** [review]
👍

**Pull request merged**